modules/hdbt_admin_tools/assets/css/menu.css

```diff
@@ -2,3 +2,7 @@
   -webkit-mask-image: url(../icons/tools.svg);
   mask-image: url(../icons/tools.svg);
 }
+.toolbar .toolbar-bar .toolbar-menu-administration .toolbar-icon.toolbar-icon-helfi-api-base-debug-settings:before {
```

Comment:
What a fun little critter 🐛 :)

Btw, we should use double colon for pseudo elements `::before`

Reply:
Good catch! Will fix.